### DISCUSSION TITLE
Add a Icons Package...!

### DISCUSSION THREAD

**Discussion by abhibhanderii** [heroui-inc/heroui#1046] [Feedback]
Hey, My name is Anonymous and I am working with **NextUI** for long time, and the biggest issue I have is The **icons**. The thing is I don't like other icons such as `font awesome, MUI, Box Icons`. But When I show the **NextUI icon's SVG code**. I fell in love with it. But I have to manually add the code in my any project.
So I highly suggest to make component based icons library like `MUI` has. 

- This is my biggest problem with **NextUI**.
- But The UI Of the Website is best....All the Framework around doesn't have this much great UI.

So, I Thank you So much for giving us this much great and easy to learn UI library...!

**Comment by adam-rocska** [comment]
yup.
+ any idea why react iconly was discontinued?

**Comment by mxvsh** [comment]
any update on this, i asked similar question

**Comment by sti-web** [comment]
Use SVGs 
This is decent web-site with SVG icons: https://www.svgrepo.com/
(500.000+ Open-licensed SVG Vector and Icons)

**Comment by iamharshdabas** [comment]
It appears that [Iconly](https://web.iconly.pro/) is the official icon library for NextUI, as mentioned in #1826. However, since Iconly is a paid service, you might consider using Solar Icons as an alternative. Below are some useful links:

- [GitHub Repository](https://github.com/480-Design/Solar-Icon-Set)
- [NPM Package](https://www.npmjs.com/package/solar-icon-set)

Additionally, Solar Icons can be used from Iconify at [iconify/solar](https://icon-sets.iconify.design/solar/), as they are required for [NextUI Pro](https://nextui.pro/documentation).

**Reply by adam-rocska** [reply]
You are awesome. Thanks!

**Reply by MisterHims** [reply]
Not exactly, many small details differ. Maybe to avoid legal issues. But well found!

**Reply by adam-rocska** [reply]
Oh, come on @MisterHims , we can’t be too picky with free stuff. Whatever the reason is, I’m totally satisfied with the results. Someone busted its ass to make that thing a reality.
